pymeasure/adapters/visa.py

```diff
@@ -181,13 +181,14 @@ def _read_bytes(self, count, break_on_termchar=False, **kwargs):
         elif break_on_termchar:
             return self.connection.read_raw(None, **kwargs)
         else:
-            read_termination = self.connection.read_termination
-            self.connection.read_termination = None
-            # Try except allows to set the read_termination even after an error.
-            try:
-                return self.connection.read_raw(**kwargs)
-            finally:
-                self.connection.read_termination = read_termination
+            result = bytearray()
+            while True:
+                try:
+                    result.extend(self.connection.read_bytes(1))
```

Comment:
Might be good to leave a comment (and reference to the upstream issue) about _why_ we are doing this.

Reply:
Done.